client/my-sites/plans/current-plan/jetpack-checklist/index.js

```diff
@@ -0,0 +1,148 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import page from 'page';
+import React, { Fragment, PureComponent } from 'react';
+import { connect } from 'react-redux';
+import { get } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Checklist from 'components/checklist';
+import getSiteChecklist from 'state/selectors/get-site-checklist';
+import QuerySiteChecklist from 'components/data/query-site-checklist';
+import Task from 'components/checklist/task';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteSlug } from 'state/sites/selectors';
+import { isDesktop } from 'lib/viewport';
+import { loadTrackingTool, recordTracksEvent } from 'state/analytics/actions';
+import { requestGuidedTour } from 'state/ui/guided-tours/actions';
+
+class JetpackChecklist extends PureComponent {
+	componentDidMount() {
+		this.props.loadTrackingTool( 'HotJar' );
+	}
+
+	isComplete( taskId ) {
+		return get( this.props.taskStatuses, [ taskId, 'completed' ], false );
+	}
+
+	handleTaskStart = ( { taskId, tourId, url } ) => () => {
+		if ( ! tourId && ! url ) {
+			return;
+		}
+
+		this.props.recordTracksEvent( 'calypso_checklist_task_start', {
+			checklist_name: 'jetpack',
+			location: 'JetpackChecklist',
+			step_name: taskId,
+		} );
+
+		if ( tourId && ! this.isComplete( taskId ) && isDesktop() ) {
+			this.props.requestGuidedTour( tourId );
+		}
+
+		if ( url ) {
+			page.show( url );
```

Comment:
Tangential (and possibly brought up before :grin: ), what about adding an `href` prop to `Task` (passed on to the button) to avoid having to call `page.show()` here?

Reply:
(Definitely not for this PR, anyway.)

Reply:
Thanks! That's a nice idea I hadn't considered 🤔